tools/validators/ontology_validator/yamlformat/validator/presubmit_validate_types_lib.py

```diff
@@ -560,8 +560,8 @@ def RunPresubmit(unmodified, modified_base, modified_client):
 
   """
 
-  findings, _ = _ValidateConfigInner(unmodified, modified_base, modified_client)
-
+  findings, _ = _ValidateConfigInner(unmodified, modified_base, modified_client, False)
+  _printFindings(findings, '')
```

Comment:
printing findings in presubmit mode is going to make a big mess because the findings will end up being printed 3x (they're already printed twice with internal presubmit).  This needs to only happen on external presubmit.  Suggest pulling the print statement into a higher level method.

Reply:
I see, will leave it in RunInteractive as it was before and taking out the one for Presubmit